samples/nextjs/src/lib/page-props-factory.ts

```diff
@@ -46,11 +47,21 @@ export class SitecorePagePropsFactory {
   private componentPropsService: ComponentPropsService;
   private dictionaryService: DictionaryService;
   private layoutPersonalizationUtils: LayoutPersonalizationUtils;
-
-  constructor() {
-    this.componentPropsService = new ComponentPropsService();
+  private layoutServiceFactory: LayoutServiceFactory;
+  private editingDataService: EditingDataService;
+
+  constructor(
+    componentPropsService: ComponentPropsService,
```

Comment:
filed + ctor + setting in ctor could be replaced with one line
private componentPropsService: ComponentPropsService

Reply:
Resolved